src/index.spec.ts

```diff
@@ -79,6 +79,10 @@ describe('ts-node', function () {
 
     it('should throw errors', function (done) {
       exec(`${BIN_EXEC} -e "import * as m from './tests/module';console.log(m.example(123))"`, function (err) {
+        if (err === null) {
```

Comment:
Since in newer Node type definitions `err` can be `null`. It was required to add check to satisfy `strictNullChecks`.